tests/test_watcher_enhanced.py

```
@@ -146,18 +158,24 @@ def test_process_new_job_stores_in_state(self, watcher_instance):
+            123, "Test Job", 10.0, "http://test.com", "Test"
+        )
 
         # Should call add_job
         watcher_instance.state.add_job.assert_called_once()
 
-    def test_process_new_job_with_min_reward_filter(self, watcher_instance, mock_config):
+    def test_process_new_job_with_min_reward_filter(
+        self, watcher_instance, mock_config
+    ):
         """Test job filtering by minimum reward."""
         mock_config.get.side_effect = lambda s, k: 50.0 if k == "min_reward" else None
```

Ruff (0.14.14)

[warning] Unused lambda argument: `s`

(ARG005)